engine/security/rootless.md

```diff
@@ -173,6 +173,8 @@ extract `docker-rootless-extras-<version>.tgz`. The archive can be extracted
 under an arbitrary directory listed in the `$PATH`. For example, `/usr/local/bin`,
 or `$HOME/bin`.
 
+You will need to [install](https://github.com/rootless-containers/rootlesskit#setup) rootlesskit.
```

Comment:
No, this is included in `docker-rootless-extras-<version>.tgz`

Reply:
It's been a few weeks and I don't recall the details, but I'm pretty sure it didn't just work for me. But I no longer have the Linux machine I was using when doing this, so I can't really help further :(